azure-pipelines.yml

```
@@ -0,0 +1,83 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+  
```

```suggestion
  pr: none
```
Pull request builds not needed as triggered on all branch commits
https://github.com/SkillsFundingAgency/das-reservations-api/blob/master/azure-pipelines.yml#L7